examples/server/server.cpp

```diff
@@ -2065,6 +2066,7 @@ static void server_print_usage(const char *argv0, const gpt_params &params,
     printf("  -v, --verbose             verbose output (default: %s)\n", server_verbose ? "enabled" : "disabled");
     printf("  -t N, --threads N         number of threads to use during computation (default: %d)\n", params.n_threads);
     printf("  -tb N, --threads-batch N  number of threads to use during batch and prompt processing (default: same as --threads)\n");
+    printf("  --threads-http N          number of threads in the http server pool to process requests (default: hardware concurrency)\n");
```

Comment:
On the doc, it's `--threads-server` but here `--threads-http`

I'd prefer the name `--threads-http` though, because saying "server" is quite unclear (easy to be confused with inference threads)